src/dvclive/live.py

```diff
@@ -269,6 +269,11 @@ def log_image(self, name: str, val):
         if not Image.could_log(val):
             raise InvalidDataTypeError(name, type(val))
 
+        if isinstance(val, str) or isinstance(val, Path):
+            from PIL import Image as ImagePIL
+
+            val = ImagePIL.open(val)
```

Comment:
I would move this to:

https://github.com/iterative/dvclive/blob/1cb31aa2f906396c1a32f1e0bd548d49af851182/src/dvclive/plots/image.py#L25-L30